src/storage/assets/assetsActions.ts

```diff
@@ -41,3 +43,19 @@ export const syncNetworkTokensInfo = createAsyncThunk('assets/syncNetworkTokensI
 
   return metadata
 })
+
+export const syncUnknownTokensInfo = createAsyncThunk(
+  'assets/syncUnknownTokensInfo',
+  async (unknownTokenIds: Asset['id'][]) => {
+    const tokens = await Promise.all(
+      unknownTokenIds.map((id) =>
+        client.node.fetchStdTokenMetaData(id).then((data) => ({
+          id,
+          ...data
+        }))
+      )
+    )
+
+    return tokens.map(({ name, symbol, decimals, id }) => ({ name, symbol, decimals, id }))
```

Comment:
What's this map for?

Reply:
To remove `totalSupply: Number256`.

Reply:
Would it be better if I wrote this?

```ts
return tokens.map(({ totalSupply, ...rest }) => ({ ...rest }))
```

Reply:
Yes, I like it better, way more explicit! 👌